src/RegistryCoordinator.sol

```diff
@@ -209,10 +220,10 @@ contract RegistryCoordinator is EIP712, Initializable, IRegistryCoordinator, ISo
             operator: msg.sender,
             operatorId: operatorId,
             quorumNumbers: quorumNumbers,
-            socket: socket
+            socket: socket,
+            operatorSignature: operatorSignature
         });
 
-        uint256 kickIndex = 0;
```

Comment:
Ah. This should stay in. `kickIndex` should be used to access `operatorKickParams` when calling `_validateChurn`. :expressionless:

Reply:
And `_deregisterOperator` below that

Reply:
Nevermind - the lists are the same length! Good callout @ypatil12 :heart: